os/KernelLibs.mk

```diff
@@ -143,6 +143,11 @@ TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_wlan$(LIBEXT)
 TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_wps$(LIBEXT)
 endif
 
+ifeq ($(CONFIG_AMEBALITE_WIFI),y)
+TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_enc$(LIBEXT)
```

Comment:
1. Could you share what the lib_enc is?
2. Do we need this with commented out?
3. Only this is not commented. Could you check?
4. Please add them in the protectedlibs.mk.

Reply:
Dear Mr Chang, lib_enc stands for encryption lib, it includes some algorithms for wifi.
In current stage, we do not need it. But in future commit, let's say next month, especially for wifi performance tuning, we might include it.

So I suggest we keep them commented out first.

And I will add them in protectedlibs.mk.